app/filters/filters.js

```diff
@@ -345,4 +345,15 @@ angular.module('portainer.filters')
   return function (createdBy) {
 	  return createdBy.replace('/bin/sh -c #(nop) ', '').replace('/bin/sh -c ', 'RUN ');
   };
-});
+})
+.filter('trimshasum', function () {
+  'use strict';
+  return function (imageName) {
```

Comment:
I believe we should just truncate (18 chars is a good value) the content of the `imageName` if it's prefixed by `sha256`.

Reply:
What I am trying do here is replicate the behavior of the docker cli. So I don't get what you mean :)

Reply:
All good then, try to replicate the behavior ;-)

Reply:
By trying I meant it is already replicating. Look at the attached screenshot.

![image](https://user-images.githubusercontent.com/4649139/34290579-88647048-e6f8-11e7-9de9-df18abafe0d1.png)

Reply:
Uh, you're right, sorry I was a bit confused as well.